platform/linux-dpdk/Makefile.am

```diff
@@ -196,6 +197,7 @@ noinst_HEADERS = \
 		  ${top_srcdir}/platform/linux-generic/include/odp_packet_socket.h \
 		  ${top_srcdir}/platform/linux-generic/include/odp_pkt_queue_internal.h \
 		  ${srcdir}/include/odp_pool_internal.h \
+		  ${top_srcdir}/platform/linux-generic/include/odp_pool_subsystem.h \
```

Comment:
I meant, the files Kevin ccreated (for ex: odp_pool_subsystem.h) already have the correct code. After the directory structure is created, we do not need to change the code in these files, they just need to be moved to their respective directories.